reactive/kubernetes_worker.py

```diff
@@ -242,6 +242,35 @@ def shutdown():
     service_stop('snap.kube-proxy.daemon')
 
 
+@when_not('config.changed.ports-to-open')
+def open_close_ports_if_needed():
+    # 1) Check which ports are open and save to already_opened
+    # opened_ports return format: "icmp" or "80/tcp"
+    already_opened = set([p.split("/")[0] for p in hookenv.opened_ports()])
+    # 2) iterate over the list from ports-to-open and check if not present to alrady_opened
+    to_open = hookenv.config().get("ports-to-open")
+    if to_open is None:
+        to_open = ""
```

Comment:
nit: this can be simplified with `to_open = hookenv.config().get("ports-to-open", "")`